### ISSUE TITLE
Python3 support

### ISSUE THREAD

**Issue by mariomulansky** [mariomulansky/PySpike#17]
Provide a Python3 version of PySpike.

**Comment by ignatenkobrain** [comment]
```python
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/brain/rpmbuild/BUILD/python-pyspike-0.3.0/python3/test/test_distance.py", line 245, in test_multi_isi
    spk.isi_distance_multi)
  File "/home/brain/rpmbuild/BUILD/python-pyspike-0.3.0/python3/test/test_distance.py", line 219, in check_multi_profile
    f_multi1 = profile_func_multi(spike_trains, [1, 2, 3])
  File "/home/brain/rpmbuild/BUILD/python-pyspike-0.3.0/python3/pyspike/isi_distance.py", line 108, in isi_profile_multi
    indices)
  File "/home/brain/rpmbuild/BUILD/python-pyspike-0.3.0/python3/pyspike/generic.py", line 66, in _generic_profile_multi
    avrg_dist = divide_and_conquer(pairs[:len(pairs)/2],
TypeError: slice indices must be integers or None or have an __index__ method
```

**Comment by mariomulansky** [comment]
Implemented in #19
Python3 support has been added to the PySpike release 0.4.
